src/components/application/Bibliomap.vue

```diff
@@ -15,13 +28,35 @@
           </v-card>
         </v-row>
       </v-col>
-      <v-col cols="12" md="4" order="1" order-md="2">
-        <v-row class="ma-2" justify="center" align="center">
-          <v-icon size="60">mdi-web</v-icon>
-          <h2>Bibliomap</h2>
-        </v-row>
-        <p class="ma-2 text text-center text-justify">{{ $t("bibliomap") }}</p>
-      </v-col>
     </v-row>
   </v-container>
 </template>
+
+<script>
+export default {
+  computed: {
+    link() {
+      if (this.$i18n.locale === 'fr') {
+        return 'https://bibliomap.inist.fr/';
+      }
+      return 'https://bibliomap.inist.fr?lang=en';
+    },
+    alignCentered() {
+      switch (this.$vuetify.breakpoint.name) {
+        case 'xs':
+          return 'center';
+        case 'sm':
+          return 'center';
+        case 'md':
+          return '';
+        case 'lg':
+          return '';
+        case 'xl':
+          return '';
+        default:
+          return '';
+      }
```

Comment:
We could compact this switch

```suggestion
      switch (this.$vuetify.breakpoint.name) {
        case 'xs':
        case 'sm':
          return 'center';
        case 'md':
        case 'lg':
        case 'xl':
        default:
          return '';
      }
```